lib/Core/EpisodeAction/EpisodeActionSaver.php

```diff
@@ -51,42 +62,59 @@ public function saveEpisodeActions(array $episodeActionsArray, string $userId):
 
     private function convertTimestampToUnixEpoch(string $timestamp): string
     {
-        return DateTime::createFromFormat(self::DATETIME_FORMAT, $timestamp)
-            ->format("U");
+        $dateTime = new DateTime($timestamp);
```

Comment:
Should you set the time zone to UTC here in case a client doesn't give one? That's what caused the problems with the other sync servers

Reply:
Hmm... Yes, if the clients don't specify their time zone, this could lead to problems.
We can just always assume UTC and always specify UTC when responding with ISO-8601 timestamps, that should fix most issues. Done this in #188

Reply:
Thanks!